File: C#/1002 - Área do Círculo.cpp
// David "Dede" Ribeiro - 2021 - @ddavidi_

using System;

class URI {
    
  static void Main(string[] args) {
      
    const double PI = 3.14159;
    double raio = Convert.ToDouble(Console.ReadLine());
    double area = PI * Math.Pow(raio, 2);
    
    Console.WriteLine("A={0:F4}", area);
  }
}

File: Java/1002 - Área do Círculo.java
/* 
   ==UserScript==
 @name         Área do Círculo
 @namespace    https://github.com/Ddavidi/uri-online-judge-beecrowd
 @description  URI 1002 - Área do Círculo
 @copyright    @ddavidi_
 @author       @ddavidi_	
   ==/UserScript==
*/

import java.util.Locale;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
    	Locale.setDefault(Locale.US);
        
        Scanner input = new Scanner(System.in);
        
        double area, pi, raio;
        
        pi = 3.14159;
        raio = input.nextDouble();
        area = pi * (raio*raio);
        
        System.out.printf("A=%.4f\n", area);
        
    }
}
